ci/scripts/collect-sources-analysis.sh

```diff
@@ -18,8 +16,6 @@ TARGET="$5"
 if [[ "$MODE" == "PR" && -n "$6" && -n "$7" ]]; then
   PR_BRANCH="$6"
   PR_KEY="$7"
-else
```

Comment:
Missing $6 or $7 may silently fail the pipeline.
At least a message should be displayed to help identify the cause